src/api/app/views/webui2/webui/search/_results_owner.html.haml

```diff
@@ -0,0 +1,41 @@
+.mt-4#search-results
+  %h4 Results:
+  - results.each do |result|
+    .search_result.mt-3
+      %h6
+        Responsible for package
+        \#{params['search_text']}
+        for #{result.rootproject} set in
+        \#{project_or_package_link project: result.project, package: result.package, short: false}
+
+      %ul.list-unstyled
+        - records = result.users.merge(result.groups) { |_role, user, group| user + group }
+        - records.each do |role_with_records|
+          %li.mt-3
+            :ruby
+              role = role_with_records.first
+              emails = role_with_records.second.map(&:email).reject(&:blank?).join(', ')
+            = role.capitalize.pluralize
+            - if emails.present? && User.session
+              = mail_to(emails) do
+                %i.far.fa-envelope.ml-1{ title: "Send email to #{role.pluralize}" }
+            .d-flex.flex-wrap.flex-column.flex-md-row
+              - role_with_records.second.each do |record|
+                .card.m-1.p-2.search-result
+                  .row.no-gutters
+                    .col-auto
+                      = link_to(user_or_group_show_path(record)) do
+                        = image_tag_for(record, size: 60, custom_class: 'align-self-center shadow-lg border border-secondary rounded border-2')
+                    .col
+                      .card-body.p-0.pl-2
+                        %ul.list-unstyled
+                          - name = record.try(:realname) || record.name
+                          %li{ title: name }
+                            = truncate(name, length: 40)
+                          - if record.class.name == 'User'
+                            %li.small
+                              = record.login
```

Comment:
If the `realname` is not set you are render `login` twice

Reply:
Good catch, I'll add a check if `realname` is present.